MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -70,21 +74,22 @@ open class EventsManager: NSObject {
      When set to `false`, flushing of telemetry events is not delayed. Is set to `true` by default.
      */
     @objc public var delaysEventFlushing = true
-    
-    func start() {
+
+    public func start() {
```

Comment:
Instead of making this public, let's take care of this internally when the NavigationEventsManger is initialized. This is also supported by requiring that the mobileEventsManager be passed at initialization time and not mutated thereafter /cc @frederoni